src/software/SfM/main_openMVG2ABC.cpp

```diff
@@ -0,0 +1,168 @@
+#include <cstdlib>
+
+// Command line
+#include "third_party/cmdLine/cmdLine.h"
+
+// OpenMVG
+#include "openMVG/sfm/sfm.hpp"
+#include "openMVG/numeric/numeric.h"
+#include "openMVG/geometry/pose3.hpp"
+using namespace openMVG::sfm;
+
+// Alembic
+#include <Alembic/AbcGeom/All.h>
+#include <Alembic/AbcCoreHDF5/All.h>
+using namespace Alembic::Abc;
+namespace AbcG = Alembic::AbcGeom;
+using namespace AbcG;
+
+
+int main(int argc, char **argv) {
+   
+    // Get arguments
+    CmdLine cmdLine;
+    std::string sfmDataFilename;
+    std::string sOutAlembic = "";
+
+    cmdLine.add(make_option('i', sfmDataFilename, "sfmdata"));
+    cmdLine.add(make_option('o', sOutAlembic, "outfile"));
+
+    try {
+        if (argc < 4) throw std::string("Invalid number of parameters in the command line.");
+        cmdLine.process(argc, argv);
+    } catch(const std::string &s) {
+        std::cout << "openMVG to alembic" << std::endl;
+        std::cout << "Usage: " << argv[0] << '\n'
+        << "[-i|--sfmdata filename, the SfM_Data file to convert]\n"
+        << "[-o|--outfile path]\n"
+        << std::endl;
+        std::cerr << s << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    //
+    SfM_Data sfm_data;
+    if (!Load(sfm_data, sfmDataFilename, ESfM_Data(ALL))) {
+        cout << "Error: The input project file \""<< sfmDataFilename << "\" cannot be read." << std::endl;
+        return EXIT_FAILURE;
+    }
+  
+    // Open Alembic archive
+    // TODO: decide wether we want to keep HDF5 or switch to Ogawa 
+    OArchive archive(Alembic::AbcCoreHDF5::WriteArchive(), sOutAlembic);
```

Comment:
Yes, this is definitely a good idea and a minor change. For the record, I am not quite sure if all the softwares used in post production can already read ogawa files, I still have to double check. So I guess it would be better to export hdf5 files by default for now.